yr/tutorials/Number_Guessing_Game.md

```diff
@@ -0,0 +1,483 @@
+
+Finally, we want to define what happens if **myGuessSlot** is not bigger or less than **secret** (equal to the value of **secret**). To do so, we need to define the **else** statement). 
+
+## Define the Intent Handler for *makeGuessIntent* (Part 4)
+
+If our guess is correct and equal to secret, we no longer need to continue the conversation. Therefore, instead of using **ask**, we will use **say** instead. We want Alexa to **say**: "(myGuessSlot) is correct." To do this, we need to use the **say** block from the **Voice** drawer, the **myGuessSlot.value** block, and the <span class = "text">join</span> block from the **Text** drawer. Try it yourself! (Hint: How many parts are there in Alexa's responses? Do we need to add parts to the <span class = "text">join</span> block?)
+
+Our final result should look like this:
+
+![Result](../images/number_guessing_game/image21.png){:.enlargeImage}
+<p style="padding-bottom: 7px"></p>
+
+Let's summarize the logic flow of our <span class = "control"> conditional if/then/else if/then/else</span> block to better understand conditionals. First, we check whether the **if** statement is true: **myGuessSlot > secret**. If it is, then whatever is in the first **then** statement is run, and the other following statements are ignored. However, if the **if** statement is false, then we move on to checking whether the **else if** statement (**myGuessSlot < secret**) is true or not. If it is true, then the second **then** statement is run and the other following statements are ignored. If it is not true, then we move onto the **else** statement and run it.
+
+Our new guessing game host Skill is now finished! Let's send it to Amazon and test it out.
+
+# Test Your Skill! 
+
+## Send the Code to Amazon
+
+At the bottom of the testing box, click on **Send Updates**. 
+
+![Send updates](../images/number_guessing_game/image33.png){:.enlargeImage}
+<p style="padding-bottom: 7px"></p>
+
+The updates should take about 1-2 minutes to finish updating. If you decide to make any changes to your Alexa Skill after completing this tutorial (e.g. updating/adding/deleting blocks), make sure you do this step again to send your new changes to Amazon! 
+
+## Test your Skill
+
+If you have an Alexa-enabled device handy, say *“Alexa, tell guessing game host to quiz me”* to it and hear the response. You can also do this on your mobile device by downloading the free **Amazon Alexa** app, which comes with a built-in Alexa simulator.
+
+If you don’t have an Alexa-enabled device on hand, the Testing page in App Inventor allows you to simulate an Alexa using your custom Alexa skill right in your browser! Simply type in the textbox at the bottom, *“tell guessing game host to quiz me”* (You can omit the Alexa at the beginning of the phrase when using the testing box). Does the conversation flow like we planned it to? Here’s an example conversation:
+
+![Example Conversation](../images/number_guessing_game/image43.png){:.enlargeImage}
```

Comment:
Maybe mention that since the "secret" number is randomly generated that the user will see different sequences than what the example is shown here. I know this is obvious but sometimes it 'might' confuse some people

Reply:
Fixed this in new merge!